presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-kms-hdfs-no-impersonation/hive-additional-site.xml

```diff
@@ -0,0 +1,6 @@
+<configuration>
+    <property>
+        <name>dfs.client.key.provider.cache.expiry</name>
+        <value>0</value><!-- disables cache -->
```

Comment:
Caches can have two problems:

- it can contain a stale entry, which would prevent us from reaching KMS server (this is addressed by forcing cache expiration)
- it can contain a valid entry which current thread would not be otherwise able to calculate (e.g. you can read from Parquet only after you read from TEXTFILE table) -- this is being addressed here

I will add comment in the code.